### DISCUSSION TITLE
o2m and m2m field display

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#7400] [Archive]
Hello!
How to display field values from another table in records view?

For example:
I have two tables: Books and Authors
Books' fields: 
- id,
- title
- author_id

Authors:
- id
- last_name

I want to show table Books like this:
````
id      title                    author_id
1       Stranger than fiction    J. Doe, W. Potter
2       C++ for dummies          S. Miller 
````

**Comment by rijkvanzanten** [comment]
Use the `related-values` display 🙂 That allows you to setup how to display the related item's content on the other end

**Reply by IlyaPetrovM** [reply]
Oh! It's actual since last release, right? Splendid!

**Reply by IlyaPetrovM** [reply]
Well, `related-values` display looks different. It indicates "2 items", but i need raw text with comma-separated values from another collection.
![image](https://user-images.githubusercontent.com/6277090/129444267-3a1422b3-8791-4690-8278-5e457bfbf316.png)

**Reply by benhaynes** [reply]
You will get the values if you hover over that "2 items". If you want something more specific for your use case, you can create a custom Display.

**Reply by IlyaPetrovM** [reply]
Ok, i'm new to vue, but I'll try)

**Reply by IlyaPetrovM** [reply]
@benhaynes 
I've noticed [there](https://github.com/directus/directus/blob/e53414671c2c539623e7d98882ccd61842062505/app/src/displays/related-values/related-values.vue#L35), that you use [utils](https://github.com/directus/directus/tree/main/app/src/utils) to compute related values


How to import this module to my custom display project?

`import getRelatedCollection from '@/utils/get-related-collection';`  
is not working obviously

**Reply by IlyaPetrovM** [reply]
Answered here: https://github.com/directus/directus/discussions/7457

**Reply by IlyaPetrovM** [reply]
@benhaynes I made custom display! :)
May be it will be useful for others.
https://github.com/IlyaPetrovM/directus_plain_text_relational_display

Examples
Admin page:
![image](https://user-images.githubusercontent.com/6277090/130299286-5e41f5e3-e06c-4a68-83d9-225fe15db77c.png)

Collection display:
![image](https://user-images.githubusercontent.com/6277090/130299430-d21fa7ee-4626-4f9e-81fe-4d17825a9c7b.png)

Editing Item:
![image](https://user-images.githubusercontent.com/6277090/130299452-935896e0-f22e-4323-a81c-a4e08f6e763b.png)

**Reply by benhaynes** [reply]
Awesome!! Can't wait to check it out... ❤️
